app/main/controllers/activity/RTMediaBuddyPressActivity.php

```diff
@@ -198,6 +200,37 @@ function bp_get_activity_parent_content( $content ) {
 		return $content;
 	}
 
+	/**
+	 * This function will check for the media file attached to the actitvity and accordingly will set type.
+	 *
+	 * @param string $type Type of the Activity.
+	 * @return string      Filtered value of the activity type.
+	 */
+	public function bp_activity_type_before_save( $type ) {
+
+		$rtmedia_attached_files = filter_input( INPUT_POST, 'rtMedia_attached_files', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+		if ( is_array( $rtmedia_attached_files ) ) {
```

Comment:
Could we use `! empty` here? check all other places.

Reply:
Sure :+1:

Reply:
Done :+1: